### PR TITLE
update tutorials

### PR DESCRIPTION
**Pull request by Songyuanwei**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: docs/cn/tutorials/advanced_train.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  clip_grad: True
+  clip_norm: 5.0
+  ema: True
+  ema_decay: 0.9999
+```
+
+#### 梯度累积
+
+梯度累积可以有效解决内存限制问题，允许**使用大的全局批量进行训练**。可以通过在yaml配置中将`train.gradient_accumulation_steps` 设置为大于1的值来使用梯度累积功能。
+以下为计算公式：
```
Comment (SamitHuang) [inline docs/cn/tutorials/advanced_train.md]: 启用后，等效的全局batch为：

**Review by SamitHuang — APPROVED** [review]

File: docs/en/tutorials/distribute_train.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# Distributed parallel training
+
+This document provides a tutorial on distributed parallel training.
+There are two ways to train on the Ascend processor, running scripts through OpenMPI or configuring `RANK_TABLE_FILE` for training.
+On GPU processors, scripts can be run through OpenMPI for training.
+
+## OpenMPI running scripts
+
+Currently, MindSpore also supports running scripts through OpenMPI's `mpirun` on Ascend hardware platform. Users can refer to [dbnet readme](../../../configs/det/dbnet/README_CN.md#34-训练) for training. The following are command use cases:
```
Comment (hadipash) [inline docs/en/tutorials/distribute_train.md:9]: insert link to English readme here

Comment (Songyuanwei) [reply docs/en/tutorials/distribute_train.md]: fixed

**Review by SamitHuang — APPROVED** [review]

**Review by hadipash — APPROVED** [review]

File: docs/en/tutorials/distribute_train.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+      python -u tools/train.py --config configs/rec/crnn/crnn_resnet34_zh.yaml &> ./train.log &
+    else
+      echo 'not 0'
+      python -u tools/train.py --config configs/rec/crnn/crnn_resnet34_zh.yaml &> /dev/null &
+    fi
+done
+```
+When training other models, simply replace the yaml config file path in the script, i.e. `path/to/model_config.yaml`.
+
+After the training has started, and you can find the training log in `train.log` under `ckpt_save_dir` specified in the config file.
```
[the remark this answers is not in the document]
Comment (Songyuanwei) [reply docs/en/tutorials/distribute_train.md:121]: `train.log` saving training logs， beacuse the training command is `python -u tools/train.py --config configs/rec/crnn/crnn_resnet34_zh.yaml &> ./train.log &`

**Review by HaoyangLee — APPROVED** [review]

**Pull request merged**